owncloudApp/src/main/java/com/owncloud/android/ui/activity/FolderPickerActivity.java

```diff
@@ -509,4 +530,8 @@ private void refreshList(boolean ignoreETag) {
             }
         }
     }
+
+    public enum ACTION_BUTTON {
```

Comment:
Adding an enum to do this, could make sense. But don't forget that enums are classes, so we should follow the convention and not use TRAILING_COMMA.

BTW, ActionButton is kind of generic, I would choose a more descriptive name, for example, PickerMode or something like that. Up to you 👍